scalyr_agent/tests/scalyr_client_test.py

```diff
@@ -161,6 +161,24 @@ def test_set_position_with_log_and_thread(self):
 
         request.close()
 
+    def test_set_log_line_attributes(self):
```

Comment:
Also have a test case where the base event has a different an attribute named `source`.